### ISSUE TITLE
Logrotate doesn't work on Ubuntu 14.04

### ISSUE THREAD

**Issue by xvillaneau** [inkscope/inkscope-packaging#7]
I am having issues with logrotate configuration on Ubuntu 14.04 with the latest packages, which are that it basically doesn't work. I assume this issue also affects all Debian-based distributions. Here are my findings :

1 - The files `/etc/logrotate.d/inkscope` and `/etc/logrotate.d/cephrestapi` are ignored by logrotate if their owner is not defined as `root:root`. It seems this is not the case in the packages:
```
% dpkg -c inkscope-common_1.1.0-0.deb | grep logrotate
drwxr-xr-x emv/emv           0 2015-02-05 16:54 ./etc/logrotate.d/
-rw-r--r-- emv/emv         117 2015-02-05 16:49 ./etc/logrotate.d/inkscope
```
For reference:
```
% dpkg -c ceph_0.87-1trusty_amd64.deb | grep logrotate
drwxr-xr-x root/root         0 2014-10-29 19:45 ./etc/logrotate.d/
-rw-r--r-- root/root      1235 2014-10-29 19:45 ./etc/logrotate.d/ceph
```

2 - There seems to be a mistake in the configuration of both inkscope and cephrestapi. The logrotate is as follows:
```
/var/log/ceph/ceph_rest_api {
    rotate 2
    daily
    compress
    sharedscripts
    postrotate
    missingok
endscript
}
```
The  `postrotate` and `endscript` instructions are not used properly and should be removed. In this case, they are indicating that `missingok` is a shell instruction executed after the rotate operation, and not a configuration statement.

**Comment by A-Dechorgnat** [comment]
part 1 fixed in 1.2.0-0

**Comment by A-Dechorgnat** [comment]
fixed
